### PR TITLE
Add Usage Information to Readme

### PR DESCRIPTION
**Pull request by ernestii**
Add the following information to readme:

## Usage

| Command | Description |
| - | - |
| `logdna register [email]` | Register a new LogDNA account |
| `logdna ssologin` | Log in to a LogDNA via single sign-on |
| `logdna login [email]` | Log in to LogDNA |
| `logdna tail [options] [query]` | Live tail with optional filtering. Options include `-h`, `-a`, `-l`, `-t` to filter by hosts, apps, levels or tags respectively. Run `logdna tail --help` to learn more. |
| `logdna switch` | If your login has access to more than one account, this command allows you to switch between them |
| `logdna search [options] [query]` | Basic search with optional filtering. Run `logdna search --help` for options. |
| `logdna info` | Show current logged in user info |
| `logdna update` | Update the CLI to the latest version |

### Examples

```sh
# Register
$ logdna register user@example.com
$ logdna register user@example.com b7c0487cfa5fa7327c9a166c6418598d # use this if you were assigned an Ingestion Key

# Login
$ logdna login user@example.com

# Tail
$ logdna tail '("timed out" OR "connection refused") -request'
$ logdna tail -a access.log 500
$ logdna tail -l error,warn

# Search
$ logdna search "logdna cli" -a logdna.log -t tag1,tag2 -n 300
$ logdna search "logdna" --from 1541100040931 --to 1541102940000

# Other
$ logdna info
$ logdna update

# Switch organization
$ logdna switch

> 1: ACME (active)
> 2: Strickland Propane
> Choose account [1-2]:
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -112,6 +112,43 @@ grunt mac
...
+## Usage
+
+| Command | Description |
+| - | - |
+| `logdna register [email] [key]` | Register a new LogDNA account. `[key]` is optional and will autogenerate. |
+| `logdna ssologin` | Login to a LogDNA user account via single sign-on. |
+| `logdna login [email]` | Login to a LogDNA user account. |
+| `logdna tail [options] [query]` | Live tail with optional filtering. Options include `-h`, `-a`, `-l`, `-t` to filter by hosts, apps, levels or tags respectively. Run `logdna tail -h` to learn more. |
+| `logdna switch` | Switch between multiple accounts if your login has access to more than one. |
+| `logdna search [options] [query]` | Limited search functionality with optional filtering (beta). See `logdna search --help`. |
```
Comment (smusali) [inline README.md]: `beta`?

Comment (ernestii) [reply README.md]: Idk, it was in `logdna -h`.

File: README.md [hunk 2]
```diff
@@ -112,6 +112,43 @@ grunt mac
 This will output the `pkg` file to the root of the repo. Signing will likely fail since we typically sign it with our Apple Developer key, but the package should still be usable, just unsigned.
 
 
+## Usage
+
+| Command | Description |
+| - | - |
+| `logdna register [email] [key]` | Register a new LogDNA account. `[key]` is optional and will autogenerate. |
```
Comment (dchai76) [inline README.md]: I think `key` could use better explanation. Like, reading this I don't know what it's for and why you would ever specify it. Maybe call it `[ingestion_key]` to start and say "If you are registering for a new account and don't yet have an ingestion key, this command will create one for you."

File: README.md [hunk 3]
```diff
@@ -112,6 +112,43 @@ grunt mac
...
+| `logdna ssologin` | Login to a LogDNA user account via single sign-on. |
```
Comment (dchai76) [inline README.md]: Let's say `Log in to LogDNA via single sign-on`

File: README.md [hunk 4]
```diff
@@ -112,6 +112,43 @@ grunt mac
...
+| `logdna login [email]` | Login to a LogDNA user account. |
```
Comment (dchai76) [inline README.md]: `Log in to LogDNA`

File: README.md [hunk 5]
```diff
@@ -112,6 +112,43 @@ grunt mac
...
+| `logdna tail [options] [query]` | Live tail with optional filtering. Options include `-h`, `-a`, `-l`, `-t` to filter by hosts, apps, levels or tags respectively. Run `logdna tail -h` to learn more. |
```
Comment (dchai76) [inline README.md]: This is kind of confusing - are we re-using `-h` for both hosts filtering and help instructions? Maybe help should be `--help`?

File: README.md [hunk 6]
```diff
@@ -112,6 +112,43 @@ grunt mac
...
+| `logdna switch` | Switch between multiple accounts if your login has access to more than one. |
```
Comment (dchai76) [inline README.md]: `If your login has access to more than one account, this command allows you to switch between them.`

File: README.md [hunk 7]
```diff
@@ -112,6 +112,43 @@ grunt mac
...
+| `logdna search [options] [query]` | Limited search functionality with optional filtering. See `logdna search --help`. |
```
Comment (dchai76) [inline README.md]: "Basic search with optional filtering. Run `logdna search --help` for options."

File: README.md [hunk 8]
```diff
@@ -112,6 +112,43 @@ grunt mac
...
+| `logdna info` | Show current logged in user info. |
+| `logdna update` | Update CLI to latest version. |
```
Comment (dchai76) [inline README.md]: "Update the CLI to the latest version."

File: README.md [hunk 9]
```diff
@@ -112,6 +112,43 @@ grunt mac
...
+$ logdna register user@example.com b7c0487cfa5fa7327c9a166c6418598d # use this if you were assigned an Ingestion Key
+
+# Login
+$ logdna login user@example.com
+
+# Tail
+$ logdna tail '("timed out" OR "connection refused") -request'
+$ logdna tail -a access.log 500
+$ logdna tail -l error,warn
+
```
Comment (dchai76) [inline README.md]: Can we get an example of how to use `logdna switch` ?

File: README.md (hunk 2 above)
Comment (ernestii) [reply README.md]: Looks like we removed support for the `key` param some time ago. I'll remove it from here. You can check commit 0c9781a70 on ldapi.

File: README.md (hunk 5 above)
Comment (ernestii) [reply README.md]: Yes, it should be `--help`

File: index.js [hunk 10]
```diff
@@ -20,13 +20,12 @@ process.title = 'logdna';
 program._name = 'logdna';
 program
     .version(pkg.version, '-v, --version')
-    .usage('[commands] [options]\n\n  This CLI duplicates useful functionality of the LogDNA web app.')
-    // .description('This CLI duplicates useful functionality of the LogDNA web app.')
+    .usage('[commands] [options]\n\nThe LogDNA CLI allows you to signup for a new account and tail your servers right from the command line.')
```
[the remark this answers is not in the document]
Comment (ernestii) [reply index.js]: done

**Pull request merged**